scripts/README.md

```diff
@@ -0,0 +1,45 @@
+# `scripts` directory
+
+This directory contains some code to build stuffs.
+
+## `babel-plugin-transform-carbon-feature-flags.js`
```

Comment:
I might be misunderstanding here, but what is this transform doing? It feels like if the feature flag is false then it would be DCE'd by uglify but I could be wrong here.

Reply:
This Babel plugin allows you to replace feature flags with the given values. This is same as [WebPack's `DefinePlugin`](https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin) in that sense, however, `DefinePlugin` works only with non-referenced variables, whereas our code using feature flags are written in a manner referring to ESM imports from `featureFlags.js`.

Reply:
Totally! What's the difference in the resulting bundle when using the babel plugin versus going through a build process without it?

Reply:
Good question - Let's use the example in this document (assuming `aFeatureFlag` equals to `true` - focus on the child nodes in the `<div>`):

# In

```javascript
 import { aFeatureFlag } from '/path/to/featureFlags';
 ...
 const MyComponent = props => (<div {...props}>{aFeatureFlag ? 'foo' : 'bar'}</div>);
 ```

# Out _without_ the Babel plugin (Exactly same as "In")

```javascript
 import { aFeatureFlag } from '/path/to/featureFlags';
 ...
 const MyComponent = props => (<div {...props}>{aFeatureFlag ? 'foo' : 'bar'}</div>);
 ```

# Out _with_ the Babel plugin

```javascript
 import { aFeatureFlag } from '/path/to/featureFlags';
 ...
 const MyComponent = props => (<div {...props}>{'foo'}</div>);
 ```

Reply:
Also, you can see the difference by changing a component to use feature flags and looking at the resulting Rollup bundle (in our `umd` directory after you run `yarn build`)

Reply:
Went ahead with removing the Babel plugin for now.